konflux/konflux-jira-integration/scripts/compliance.sh

```diff
@@ -887,8 +927,11 @@ for line in $components; do
 
     # Retrigger component if build failed and --retrigger flag is set
     if [[ "$retrigger" == "true" ]]; then
+        # Skip retriggering for components with skipped null status
```

Comment:
Seems like we should also not retrigger components that have a `null` status but no exception? There'd be nothing to retrigger in that case, right?